src/Flow.php

```
@@ -61,6 +67,21 @@ public function run(string|Closure $action): Flow
 
+    /**
+     * @param class-string<ValidatingStep>|ValidatingStep $action
+     * @return $this
+     */
+    public function validate(string|ValidatingStep $action): Flow
+    {
+        if (is_string($action)) {
+            $action = resolve($action);
+        }
+
+        $this->steps[] = new ValidationStep($action);
```

PHPStan (2.1.33)

Property JustSteveKing\Flows\Flow<TPayload>::$steps (array<int, class-string<JustSteveKing\Flows\Contracts\FlowStep>|Closure>) does not accept non-empty-array<int, class-string<JustSteveKing\Flows\Contracts\FlowStep>|Closure|JustSteveKing\Flows\Steps\ValidationStep>.


(assign.propertyType)